api/lib/domain/services/authentication/fwb-oidc-authentication-service.js

```diff
@@ -15,6 +15,7 @@ class FwbOidcAuthenticationService extends OidcAuthenticationService {
       clientId: config[configKey].clientId,
       clientSecret: config[configKey].clientSecret,
       configKey,
+      extraAuthorizationUrlParameters: config[configKey].acrValues && { acr_values: config[configKey].acrValues },
```

Comment:
```suggestion
      extraAuthorizationUrlParameters: { acr_values: config[configKey].acrValues },
```
Comme `config[configKey].acrValues` n'est pas un objet cela risque de créer une erreur sur ces lignes de code

```javascript
if (this.extraAuthorizationUrlParameters) {
  Object.assign(authorizationParameters, this.extraAuthorizationUrlParameters);
}
```